app/src/test/java/edu/nextstep/camp/calculator/MainPresenterTest.kt

```diff
@@ -122,6 +122,25 @@ internal class MainPresenterTest {
         assertThat(exceptionSlot.captured).isInstanceOf(IllegalArgumentException::class.java)
     }
 
+    @Test
+    @DisplayName("calculate 한 결과가 리스트 형태로 보여진다.")
+    fun calculateHistoryTest() {
+        val presenterHasInitExpression = MainPresenter(mockView, expression = Expression(listOf(12, Operator.Plus, 3)))
```

Comment:
```suggestion
        val presenterWithExpression = MainPresenter(mockView, expression = Expression(listOf(12, Operator.Plus, 3)))
```
변수명이 명사구가 아닌 문장으로 보이네요.
전치사를 적절하게 활용해도 좋을 것 같아요.